packages/pieces/src/lib/apps/todoist/actions/create-task-action.ts

```diff
@@ -50,7 +50,7 @@ export const todoistCreateTaskAction = createAction({
     assertNotNullOrUndefined(token, 'token');
     assertNotNullOrUndefined(content, 'content');
 
-    return await todoistClient.tasks.create({
+    return await todoistRestClient.tasks.create({
```

Comment:
Please rename projectId -> project_id here as well.

Reply:
fixed in 07490b4